beps/0005-split-backend-discovery/README.md

```diff
@@ -209,7 +278,7 @@ We also propose adding a new method to this service to give a list of plugins ac
 ```ts
 interface DiscoveryService {
   ...
-  listPlugins: () => Promise<string[]>;
+  listFeatures: () => Promise<BackstageFeature[]>;
```

Comment:
Who would consume this list? Some management plugin or so? Wondering if this should not be here at all, but instead have a `GatewayClient` that has this method or whatever. Not sure that the discovery _interface_ should deal with this list at all, but the _implementation_ might for sure.

Reply:
Good point, I'll adjust.